BraveRewardsUI/Settings/SettingsViewController.swift

```diff
@@ -137,14 +137,13 @@ class SettingsViewController: UIViewController {
   @objc private func rewardsSwitchValueChanged() {
     state.ledger.isEnabled = settingsView.rewardsToggleSection.toggleSwitch.isOn
     updateVisualStateOfSections(animated: true)
+    NotificationCenter.default.post(name: .adsOrRewardsToggledInSettings, object: nil)
```

Comment:
I know we reverted on this @kylehickinson, but resetting the status of NTP notification should happen only after toggling Rewards state in settings

otherwise tapping on Rewards button from ntp notification will trigger second notification, but we only want one
this solution should be enough, even if there are other ways/places to toggle Rewards

Reply:
Im not really in love with this solution mostly because it looks like we're purposely missing adding notification posts to meet the spec of this feature when we should be handling it on Client side, but I'm fine with it considering the naming change to the notification defining that its happening `in settings`